VERSION

```diff
@@ -1 +1 @@
-8.2.0
+8.3.0
```

Comment:
As we are already on 8.3.1 with the latest release it appears the file was outdated even before you changed it. The correct version is 8.4.0 🙂